ember-cli-build.js

```diff
@@ -73,5 +75,10 @@ module.exports = function() {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
-  return app.toTree();
+  var bowerTree = funnel('bower_components', {
```

Comment:
Not sure how to both do that and make it possible for the files to be overridden. Any ideas?